app/electron/main.ts

```diff
@@ -342,7 +338,11 @@ function startElecron() {
         protocol: 'file:',
         slashes: true,
       });
-    const { width, height } = calculateWindowSize();
+
+    // WSL has a problem with full size window placement, so make it smaller.
+    const windowMargin = isWSL();
+    const { width, height } = windowSize(screen.getPrimaryDisplay().workAreaSize, windowMargin);
```

Comment:
Nit: Reading this, I'd assume windowMargin is a numeric value. It's good practice for booleans to be indicated as such by  using `is` or `with` or similar prefixes.

Reply:
Nit squashed withThanks.